### ISSUE TITLE
When importing allow pushing without build

### ISSUE THREAD

**Issue by LecrisUT** [juhp/fbrnch#56]
As a complete opposite to #55, during the `fbrnch import` I would want in my workflow to stop after `git push` and before `koji build` so that I run `fbrnch parallel` instead, or run the build only for rawhide. Maybe this can work together with #51 where you can have a few template configs in the top-level folder (where `fbrnch import` is run, e.g. `~/FedoraRPMs`) and then we can choose the appropriate config like the `mock -r` approach.

**Comment by juhp** [comment]
Okay I am wish you about not building immediately after import, but again why do you need to push then if you are not building. Maybe also ties to #66

**Comment by LecrisUT** [comment]
Yes, this one is about having some other updates already prepared in a side-tag. I guess it would also be important to have a way to pull in side-tags to use for `mock` :thinking:

**Comment by juhp** [comment]
Okay well I can certainly make push available after import - that is reasonable.

(Mock seems like a separate topic)
